12-http/example3/main.go

```diff
@@ -18,20 +21,46 @@ type user struct {
 	Phone string
 }
 
-// users is a slice of users.
-var users []user
+var (
+	// mu protects users.
+	mu sync.RWMutex
+	// initialize users, so that the output JSON won't be null
+	// prior to adding users.
+	users = make([]user, 0)
+)
```

Comment:
Only if they are related and the var can be the comment for everything. These are not variables that could be grouped together under a single comment.